packages/icons/.env

```diff
@@ -1,6 +1,10 @@
-FIGMA_FETCHER_FILE_KEY="KEY_EXAMPLE"
-FIGMA_FETCHER_FIGMA_TOKEN="figd_EXAMPLE"
-FIGMA_FETCHER_PAGE_NAMES="Actions, Arrows"
+# figma file key, should be set in .env.local
+FIGMA_FETCHER_FILE_KEY=SeWLzXxzzYfvfjzzbTYUl1
```

Comment:
This looks like real key, please use fake one

Reply:
this key - just figma id, why its forbidden?

Reply:
@leonid we have this key inside https://github.com/acronis/ui-component-library/blob/main/packages/documentation/guide/onboarding.md#useful-links

I thought it's not private

Reply:
> this key - just figma id, why its forbidden?

We have security check that will throw an error on possible private keys. But yes in that case we already have in docs. So you can keep it and we will add to ignore file if needed later.